client/src/utils/ApiFetching.jsx

```diff
@@ -0,0 +1,37 @@
+export const fetchPost = (endpoint,body, headers = {Accept: "application/json","Content-Type": "application/json"}) => {
+    return fetch(endpoint,{
+        method: "POST",
+        headers,
+        body:JSON.stringify(body)
+      }).then( res => {
+        return res.json()
+      }).then((res)=>{
+        return res
+      })
+      .catch(err => {
+        console.log("Fetch error is: ", err.message);
+      })
+    
+    }
+  
+    export const fetchGet =   (endpoint, params = null , headers = {Accept: "application/json","Content-Type": "application/json"}) => {
+    
+      if (params !== null) {
+        const urlParams =  '?' + new URLSearchParams(Object.entries(params));
+        endpoint = endpoint + urlParams
+      }
+     return  fetch(endpoint,{
+        method: "GET",
```

Comment:
Notice how both this method and the one above are almost identical? I was thinking that you create a general fetch wrapper and you pass in the method type of thing

Reply:
This is in the ticket backlog.